test/eventing-kafka.bash

```diff
@@ -29,3 +29,29 @@ function upstream_knative_eventing_kafka_e2e {
   logger.info 'Starting eventing-kafka tests'
   run_e2e_tests
 }
+
+function upstream_knative_eventing_kafka_broker_e2e {
+  logger.info 'Setting Kafka as default broker class'
+
+  local root_dir
+  root_dir=$(dirname "$0")/..
+
+  # Set Kafka Broker as default Broker class
+  oc patch knativeeventing -n knative-eventing knative-eventing --patch-file "${root_dir}/test/config/eventing/kafka-broker-default-patch.yaml"
```

Comment:
my bash foo isn't working but I don't know what wrong :thinking: 
```
error: unable to read patch file: open ./test/../test/config/eventing/kafka-broker-default-patch.yaml: no such file or directory
```

Reply:
cc @mgencur @matzew I need a couple of fresh eyes to catch what's wrong

Reply:
OK, I will take a look :+1:  

Sorry but I have not seen the comment before :sweat_smile: